app/uk/gov/hmrc/hmrcemailrenderer/templates/transactionengine/TransactionEngineTemplates.scala

```diff
@@ -890,6 +908,42 @@ object TransactionEngineTemplates {
             priority = Some(MessagePriority.Urgent)
         ),
         MessageTemplate.createWithDynamicSubject(
+          templateId = "transactionEngineHMRCPSAPRACAUTHSuccess",
+          fromAddress = transactionEngineAddress,
+          service = ServiceIdentifier.PensionSchemePractitioners,
+          subject = extractSubject(_),
+          plainTemplate = psa.prac.txt.transactionEngineHMRCPSAPRACAUTHSuccess.f,
+          htmlTemplate = psa.prac.html.transactionEngineHMRCPSAPRACAUTHSuccess.f,
+          priority = Some(MessagePriority.Urgent)
+        ),
+        MessageTemplate.createWithDynamicSubject(
+          templateId = "transactionEngineHMRCPSAPRACAUTHFailure",
+          fromAddress = transactionEngineAddress,
+          service = ServiceIdentifier.PensionSchemePractitioners,
+          subject = extractSubject(_),
+          plainTemplate = psa.prac.txt.transactionEngineHMRCPSAPRACAUTHFailure.f,
+          htmlTemplate = psa.prac.html.transactionEngineHMRCPSAPRACAUTHFailure.f,
+          priority = Some(MessagePriority.Urgent)
+        ),
+        MessageTemplate.createWithDynamicSubject(
+          templateId = "transactionEngineHMRCPSAPRACDEAUTHSuccess",
+          fromAddress = transactionEngineAddress,
+          service = ServiceIdentifier.PensionSchemePractitioners,
+          subject = extractSubject(_),
+          plainTemplate = psa.prac.txt.transactionEngineHMRCPSAPRACAUTHSuccess.f,
+          htmlTemplate = psa.prac.html.transactionEngineHMRCPSAPRACAUTHSuccess.f,
+          priority = Some(MessagePriority.Urgent)
+        ),
+        MessageTemplate.createWithDynamicSubject(
+          templateId = "transactionEngineHMRCPSAPRACDEAUTHFailure",
+          fromAddress = transactionEngineAddress,
+          service = ServiceIdentifier.PensionSchemePractitioners,
+          subject = extractSubject(_),
+          plainTemplate = psa.prac.txt.transactionEngineHMRCPSAPRACAUTHFailure.f,
```

Comment:
Could you check if you are you using the right templates? In these examples the template does not match the html file:

TemplateId is transactionEngineHMRCPSAPRACDEAUTHFailure but you call 
transactionEngineHMRCPSAPRACAUTHFailure.f

TemplateId transactionEngineHMRCPSAPRACDEAUTHSuccess but you call 
transactionEngineHMRCPSAPRACAUTHSuccess.f